src/test/java/ca/gc/aafc/agent/api/entities/PersonCrudIT.java

```diff
@@ -12,15 +12,15 @@
 import org.springframework.test.context.ActiveProfiles;
 
 import ca.gc.aafc.agent.api.testsupport.factories.PersonFactory;
-import ca.gc.aafc.dina.testsupport.DBBackedIntegrationTest;
+import ca.gc.aafc.dina.testsupport.DatabaseSupportService;
 
 /**
  * Test suite to validate {@link Person} performs as a valid Hibernate Entity.
  */
 @SpringBootTest
 @Transactional
 @ActiveProfiles("test")
-public class PersonCrudIT extends DBBackedIntegrationTest {
+public class PersonCrudIT extends DatabaseSupportService  {
```

Comment:
I believe the idea here was to compose the DatabaseSupportService instead of extend it